packages/node_modules/@ciscospark/plugin-conversation/src/conversation.js

```diff
@@ -469,10 +469,16 @@ const Conversation = SparkPlugin.extend({
   /**
    * Creates a ShareActivty for the specified conversation
    * @param {Object} conversation
+   * @param {Object} activity
    * @returns {ShareActivty}
    */
-  makeShare(conversation) {
-    return ShareActivity.create(conversation, null, this.spark);
+  makeShare(conversation, activity) {
```

Comment:
It needs to create the activity using the provisional activity created at the web-client passed as the second param, else the clientTempIds won't match and would result in a broken image on the clients.